keps/sig-apps/4017-pod-index-label/README.md

```diff
@@ -411,7 +411,7 @@ existing label which other things may depend on, for example).
 - Docs are clear about what happens if two pods get the same value (it is set by workload controllers, nothing in the API system will prevent collisions from happening).
 
 #### GA
-Fix any potentially reported bugs.
+- Fix report doc bug: https://github.com/kubernetes/kubernetes/pull/125502/
 
 <!--
 **Note:** Generally we also wait at least two releases between beta and
```

Comment:
Below, in `### Version Skew Strategy`, we have:
```
and the node componets were at version N-1 where this feature does not exist
```
but should be:
```
and the node components were at version N-1 where this feature does not exist
```

Reply:
updated, thanks for the catch